packages/compass-import-export/src/components/query-viewer.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import getShellJS from '../utils/get-shell-js';
+import { Code } from '@mongodb-js/compass-components';
+
+export function QueryViewer({
```

Comment:
Is it really worth it to have a separate file for this? There are not much types here anyway

Reply:
Sorry, I should've been more clear, I read the PR description, I wanted to ask whether it's worth to spent time moving it back and forth before the file is converted to typescript, FWIW you are not getting anything much from typescript here, the interface of the component is two very vague properties covered by prop types of the parent component anyway